src/make/cards.js

```diff
@@ -275,13 +275,16 @@ function doCard(rawCard, cards, code, set) {
 
   if (rawCard.layout === 'split')
     name = rawCard.names.join(' // ')
+    
+  //separate landsfrom 0cmc cards by setting 0cmc to .2
+  var cmcadjusted = rawCard.cmc || 0.2
```

Comment:
I think this should be camelCased to be `cmcAdjusted`.

For the values `0.2` and `0.1` perhaps they should be constants listed in the same place, because otherwise the reader will have to pan around the file to find related values. `0.2` suggests that there is a partner value but someone might not realize that `0.1` has a partner value.